Sources/WalletConnectModal/Modal/Screens/GetAWalletView.swift

```diff
@@ -67,6 +66,8 @@ struct GetAWalletView: View {
     }
 }
 
+#if DEBUG
```

Comment:
Could you please double check that previews is wrapped in DEBUG everywhere?

Reply:
Added where it was missing